web/themes/joinup/templates/rdf_entity/rdf-entity--asset-release--full.html.twig

```diff
@@ -7,7 +7,12 @@
 <article{{ attributes }}>
   <div class="page__content">
     {{ content.field_isr_release_notes }}
-    <div class="version-wrapper">{{ content.field_isr_is_version_of }}</div>
```

Comment:
`field_isr_is_version_of` being removed from the template, it can be removed from the display as well.